medor_custom_stats/models/res_company.py

```diff
@@ -0,0 +1,18 @@
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    nb_subscribers = fields.Integer(compute="_compute_company_data")
+    nb_cooperators = fields.Integer(compute="_compute_company_data")
+
+    def _compute_company_data(self):
+        for company in self:
+            subscribers = (
+                self.env["res.partner"].sudo().search([("subscriber", "=", True)])
+            )
+            company.nb_subscribers = len(subscribers)
+
+            cooperators = self.env["res.partner"].sudo().search([("member", "=", True)])
```

Comment:
this has changed when `cooperator` became multi-company compatible. your line still works but it’s more of a convenience function, and works for the current company only.
```suggestion
            cooperators = self.env["cooperative.membership"].sudo().search(
                [("company_id", "=", company.id), ("member", "=", True)]
            )
```

Reply:
fixed